apps/src/code-studio/pd/professional_learning_landing/SelfPacedProgressTable.jsx

```diff
@@ -39,7 +39,7 @@ const CourseRow = ({
             {percent_completed}% {i18n.selfPacedPlCompleted()}
           </BodyThreeText>
           {/* Progress bar */}
-          <div className={styles.progressBar}>
+          <div className={styles.progressBar} data-testid="progress-bar">
```

Comment:
I thought about adding something like that! But it says "(percent)% Completed" right above it so I thought it would be redundant 😅

Reply:
oo does a screen reader read that out loud? I'm not sure if this is what you're referring to, but there is a way to add an aria-label that doesn't show, but a screen reader would still read

Reply:
Oh it doesn't! VoiceOver on Mac only reads links and buttons. I checked this on the Teacher homepage, and it does the same thing there too. I'll add a ticket for this in the next AccHack.

Reply:
Okay- it should be as simple as wrapping the text in a <label>. But yes- thanks for making a ticket!

Reply:
Neither the paragraph or progress bar is a link, so it'd need to be something else. I'm too scattered today to look into it, but will look tomorrow! This makes me excited for the Deque training, since I assumed screen readers read all the text on a page 🥲

Reply:
> use keys to navigate by header size (and drill down into reading the text directly if they choose

A ha! That makes sense, thanks for explaining. This is an interesting case…maybe the `<figure>` and `<figcaption>` elements would work? ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/figure_role)) 

Let's def brainstorm on this, and thanks for bringing it up!